src/docs/product/crons/getting-started/cli.mdx

```diff
@@ -59,6 +59,32 @@ sentry-cli monitors run my-monitor-slug -- python path/to/file.py
 sentry-cli monitors run my-monitor-slug -- node path/to/file.js
 ```
 
+### Configuring Your Monitor via the CLI
+
+You can also use the Sentry CLI to configure your cron monitor when you run your job. This way, you can avoid having to first set up the monitor through the Sentry web interface.
+
+Configure the cron monitor by providing the cron schdule in crontab format using the `--schedule` or the equivalent `-s` argument when executing the `sentry cli monitors run` command. Please make sure to enclose the schedule in quotes, so that your shell parses the argument correctly, like so:
+
+```bash
+sentry-cli monitors run --schedule "<expected schedule>" <monitor-slug> -- <command> <args>
+```
+
+When providing the `--schedule` argument, we also support the following optional arguments to provide additional configuration:
+
+- `--check-in-margin`: The allowed margin of minutes after the expected check-in time that the monitor will not be considered missed for.
+- `--max-runtime`: The allowed duration in minutes that the monitor may be in progress for before being considered failed due to timeout.
+- `--timezone` _(beta, use cautiously!)_: A tz database string (e.g. "Europe/Vienna") representing the monitor's execution schedule's timezone. Please be **absolutely sure that you provide a valid tz database string** when using this argument; as of writing, the CLI lacks the capability to validate this string, and so **the monitor creation will silently fail when the string is invalid!** For now, we recommend verifying that the cron monitor was created successfully in Sentry when using this argument.
```

Comment:
```suggestion
- `--timezone` _(beta, use cautiously!)_: A [tz database identifier string](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) (e.g. "Europe/Vienna") representing the monitor's execution schedule's timezone. Please be **absolutely sure that you provide a valid tz database string** when using this argument; as of writing, the CLI lacks the capability to validate this string, and so **the monitor creation will silently fail when the string is invalid!** For now, we recommend verifying that the cron monitor was created successfully in Sentry when using this argument.
```
Let's include a link to the tz database strings so that users have something to reference since it's important. Feel free to change the link if you have a better one!